client/apps/edit/components/section_header/index.coffee

```diff
@@ -0,0 +1,56 @@
+React = require 'react'
+moment = require 'moment'
+RichTextParagraph = React.createFactory require '../../../../components/rich_text/components/input_paragraph.coffee'
+{ div, p, textarea } = React.DOM
+
+
+module.exports = React.createClass
+  displayName: 'SectionHeader'
+
+  changeTitle: (e) ->
+    if e.key is 'Enter'
+      e.preventDefault()
+    @props.article.set 'title', this.refs.title.value
+    @props.saveArticle()
+
+  changeLeadParagraph: (html) ->
+    @props.article.setLeadParagraph(html)
+    @props.saveArticle()
+
+  getPublishDate: ->
```

Comment:
Good points - this field can probably be set as article attribute directly, setLeadParagraph may be ready to deprecate now that we've changed it to draft -- will check it out